README.md

```diff
@@ -123,7 +123,7 @@ app.foo = {
 
 ## Linting
 
-The app is ruled by [eslint](https://eslint.org/) to keep code style consistent. It has an [editorconfig](https://editorconfig.org/) file which defines some common rules to correctly configure IDE formatting.  
+The app is ruled by [eslint](https://eslint.org/) to keep code style consistent. It has an [editorconfig](https://editorconfig.org/) file which defines some common rules to correctly configure IDE formatting.
```

Comment:
```suggestion
The app is ruled by [eslint](https://eslint.org/) to keep code style consistent. It has an [editorconfig](https://editorconfig.org/) file which defines some common rules to correctly configure IDE formatting.  
```
In markdown files, the spaces at the end of line as a meaning: Go to the next line.
This one was intended to be here.
Is you use vscode, and you have enable the option "trimTrailingWhitespace" (which I think is a good idea), you can disable it for markdown files only:

```json
    "[markdown]": {
      "files.trimTrailingWhitespace": false,
    }
```